rocky/tests/conftest.py

```diff
@@ -609,3 +621,127 @@ def mock_scheduler_client_task_list(mocker):
     mock_scheduler_client_session.get.return_value = response
 
     return mock_scheduler_client_session
+
+
+@pytest.fixture
+def valid_time():
+    return datetime.now(timezone.utc)
+
+
+@pytest.fixture
+def octopoes_api_connector(request) -> OctopoesAPIConnector:
+    test_node = f"test-{request.node.originalname}"
+
+    connector = OctopoesAPIConnector(settings.OCTOPOES_API, test_node)
+    connector.session.mount("http://", HTTPAdapter(max_retries=Retry(total=3, backoff_factor=1)))
+
+    connector.create_node()
+    yield connector
+    connector.delete_node()
+
+
+def seed_system(octopoes_api_connector: OctopoesAPIConnector, valid_time):
```

Comment:
The pytest way would be to make this a fixture. Pytest will execute fixtures only once per test, so the octopoes_api_connector and valid_time of this fixture would be the same as the one the test itself get.

Reply:
I am not a big fan of using fixtures for data seeding (also see [this](https://github.com/minvws/nl-kat-coordination/pull/2045#discussion_r1402170164) discussion). Perhaps you can convince me otherwise but maybe for now this works just as good?